vms/rpcchainvm/runtime/container/client.go

```diff
@@ -0,0 +1,79 @@
+package container
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/containers/podman/v4/libpod/define"
+	"github.com/containers/podman/v4/pkg/bindings/containers"
+	"github.com/containers/podman/v4/pkg/bindings/images"
+	"github.com/containers/podman/v4/pkg/specgen"
+)
+
+var _ Container = (*Client)(nil)
+
+type Container interface {
+	// Start attempts to Start a container.
+	Start(ctx context.Context, image string) (string, error)
+
+	// Stop attempts to Stop a container.
+	Stop(ctx context.Context, id string) error
+
+	// Pull attempts to pull a container image.
+	Pull(ctx context.Context, image string) ([]string, error)
+
+	// Exists returns true if a container id exists.
+	Exists(ctx context.Context, id string) (bool, error)
+
+	// WaitForStatus will block until container meets the expected container status.
+	WaitForStatus(ctx context.Context, id string, status define.ContainerStatus) (int32, error)
+}
+
+type Client struct{}
```

Comment:
client isn't really even needed this can all be removed or maybe we keep Stop/Start